server/moodle/mod/livequiz/form/createquizform.php

```diff
@@ -0,0 +1,24 @@
+<?php
+require_once("$CFG->libdir/formslib.php");
+
+class createquizform extends moodleform {
```

Comment:
Please follow moodle standard in naming: [https://moodledev.io/general/development/policies/codingstyle](url)

Reply:
will make sure we do (and get this changed to reflect it)